doitall.sh

```diff
@@ -8,9 +8,9 @@ test -n "$proj" || proj=Factory
 test -n "$repo" || repo=standard
 
 if [ "$proj" = "Factory" ]; then
-    product=000product
```

Comment:
That feels like we break on Factory:PowerPC, Factory:ARM, Factory:zsystems

Reply:
Those use the product builder already?

Reply:
There is openSUSE/osc-plugin-factory#1112 for zsystems, but not sure what is staged vs working.

Reply:
@DimStar77 That diff only happens if diffing your pr vs. mine, so it would be already broken in master if that was the case?

Reply:
Ah, so it was meant to be the other way around, only use 000product for Factory itself but not the ports?

Reply:
as for the ttm submission: zSystems is 'in limbo' - the old prod builder doesn't produce anything usable, the new one needs more work on ttm (openSUSE/osc-plugin-factory#1112  is a beginning, but we know it won't handle the _multibuild setup created by the new prod builder service)

Reply:
I replaced it with the usual switch-case statement from the other files now, so that should be fine.

Reply:
> Ah, so it was meant to be the other way around, only use 000product for Factory itself but not the ports?

My initial pr to get started? yes, that was breaking 'my things first' - then fix and follow through to the ports.

But happy you're testing this in Leap now :)